### PR TITLE
fix: [CDS-44955]: UI breaking on adding null value to paths from YAML

### PR DESCRIPTION
**Pull request by vikrant-harness**
### Summary

- On adding null value to paths YAML in services/Manifests the UI breaks on typing something into the field.

#### Screenshots

https://user-images.githubusercontent.com/106532291/197048808-e6786a96-5aed-4764-b357-6967f51a8493.mov

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by sonaligoyal-harness — DISMISSED** [review]

**Review by Eric761 — DISMISSED** [review]

File: src/modules/70-pipeline/components/ManifestSelection/ManifestListView/AttachPathYamlFlow.tsx [hunk 1]
```diff
@@ -63,7 +63,9 @@ function AttachPathYamlFlow({
     if (manifestStore === ManifestStoreMap.Harness) {
       return valuesPaths
     }
-    return valuesPaths?.map((path: string) => ({ path, uuid: uuid(path, nameSpace()) }))
+    return valuesPaths
+      ?.filter((path: string) => !!path)
+      ?.map((path: string) => ({ path, uuid: uuid(path, nameSpace()) }))
```
Comment (vkbansal-harness) [inline src/modules/70-pipeline/components/ManifestSelection/ManifestListView/AttachPathYamlFlow.tsx:68]: Nitpick: If filter works, then map will definitely work. Hence no need for `?`

File: src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/ServerlessAwsLambdaManifest/ServerlessAwsLambdaManifest.tsx [hunk 2]
```diff
@@ -95,7 +95,9 @@ function ServerlessAwsLambdaManifest({
         paths:
           typeof specValues.paths === 'string'
             ? specValues.paths
-            : specValues.paths?.map((path: string) => ({ path, uuid: uuid(path, nameSpace()) }))
+            : specValues.paths
+                ?.filter((path: string) => !!path)
+                ?.map((path: string) => ({ path, uuid: uuid(path, nameSpace()) }))
```
Comment (vkbansal-harness) [inline src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/ServerlessAwsLambdaManifest/ServerlessAwsLambdaManifest.tsx:100]: can we abstract this to a util and reuse?

**Review by chetanvekariya-harness — DISMISSED** [review]

File: src/modules/70-pipeline/components/ManifestSelection/ManifestListView/AttachPathYamlFlow.tsx (hunk 1 above)
Comment (vikrant-harness) [reply src/modules/70-pipeline/components/ManifestSelection/ManifestListView/AttachPathYamlFlow.tsx]: done

File: src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/ServerlessAwsLambdaManifest/ServerlessAwsLambdaManifest.tsx (hunk 2 above)
Comment (vikrant-harness) [reply src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/ServerlessAwsLambdaManifest/ServerlessAwsLambdaManifest.tsx]: done

**Review by chetanvekariya-harness — APPROVED** [review]

**Review by bhavya-sinha — APPROVED** [review]

**Pull request merged**